dags.yaml

```diff
@@ -188,7 +191,7 @@ bqetl_devtools:
     retry_delay: 30m
 
 bqetl_main_summary:
-  schedule_interval: 0 2 * * *
+  schedule_interval: 15 2 * * *
```

Comment:
This will require some changes here: https://github.com/mozilla/telemetry-airflow/blob/fd54979f705b7d402b910407b79e099162b36f91/dags/attitudes_daily.py#L60

Reply:
And here https://github.com/mozilla/telemetry-airflow/blob/fd54979f705b7d402b910407b79e099162b36f91/dags/firefox_public_data_report.py#L92

Reply:
And a couple of changes in the `jetstream` DAG: https://github.com/mozilla/telemetry-airflow/blob/fd54979f705b7d402b910407b79e099162b36f91/dags/jetstream.py#L58

Reply:
And in the `kpi_forecasts` DAG https://github.com/mozilla/telemetry-airflow/blob/fd54979f705b7d402b910407b79e099162b36f91/dags/kpi_forecasts.py#L64